docs/source/general/settings.rst

```diff
@@ -189,3 +189,11 @@ Currently, ``DEFAULT_FROM_EMAIL`` is set to to ``webmaster@localhost``.
 When this setting is enabled, every accounting instance saved from the API will have its ``groupname`` attribute automatically filled in.
 The value filled in will be the ``groupname`` of the ``RadiusUserGroup`` of the highest priority among the RadiusUserGroups related to the user with the ``username`` as in the accounting instance.
 In the event there is no user in the database corresponding to the ``username`` in the accounting instance, the failure will be logged with `info` level but the accounting will be saved as usual.
+
+``CUSTOM_TYPES_FOR_NAS``
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Please note: this settings won't exist by default**
+
+This setting should have a tuple of tuples, where each tuple should be an option for `ChoiceField` for the NAS form. This setting should be used to allow custom NAS types to be selected from the admin interface
+when creating a new NAS instance. 
```

Comment:
If you could supply an example for our users it would be great, consider that many of our users are not familiar with django and some are not even familiar with python.

Reply:
PS: remember to build the docs and look at it in your browser before committing, it will help you to spot if there's any mistake or anything else to improve